pom.xml

```diff
@@ -92,13 +93,6 @@ under the License.
       <scope>provided</scope>
     </dependency>
 
-    <!-- ScriptEngines -->
-    <dependency>

```

Comment:
Btw, there is no previous version of the plugin published on Maven Central, its version is 3.0.0-SNAPSHOT which assumes interface changes, that are not finalized. Now is the time to avoid using groovy.

Reply:
For Maven the versioning scheme works a bit different. The 3.x.x means this plugin requires at least Maven 3. If we break backwords compatibility it is for the next Maven version. 
This scheme has proven itself over the years, now there's no confusion anymore if a certain plugin works with Maven 2 or not.